### ISSUE TITLE
Question: Child & Reproductive Age

### ISSUE THREAD

**Issue by mboyas-mitre** [verbal-autopsy-software/InterVA4#10]
https://github.com/verbal-autopsy-software/InterVA4/blob/792fd94fcad1f143f55865e580a4ea0bc233e5bb/InterVA4_1.7/R/InterVA.R#L373

`input.current[5]` seems to correspond to child; what is `child` defined as? It seems possible that some children may be of reproductive age and others may not.

**Comment by richardli** [comment]
You are right. child is 5 to 14 yrs old. Again, InterVA-4 logic dictates this line here.
